mmhuman3d/core/conventions/keypoints_mapping/gta.py

```diff
@@ -0,0 +1,102 @@
+GTA_KEYPOINTS = [  # idx - original name
+    'gta_head_top',  # 00  - head_top
+    'head',  # 01  - head_center
+    'neck',  # 02  - neck
+    'gta_right_clavicle',  # 03
+    'right_shoulder',  # 04  - right_shoulder
+    'right_elbow',  # 05  - right_elbow
+    'right_wrist',  # 06  - right_wrist
+    'gta_left_clavicle',  # 07
```

Comment:
In `human_data`, `left_clavicle_3dhp` and `right_clavicle_3dhp` is used
https://github.com/open-mmlab/mmhuman3d/blob/main/mmhuman3d/core/conventions/keypoints_mapping/human_data.py#L186-L187 . I think  the convention is to put the dataset as a suffix i.e. `right_clavicle_gta` - you might want to confirm this with @ttxskk 

Additionally, I think we are using `left_` and `right_` headers in the `get_flip_pairs` function so this might not work if `gta_` is used as the prefix https://github.com/open-mmlab/mmhuman3d/blob/main/mmhuman3d/core/conventions/keypoints_mapping/__init__.py#L228-L245

Reply:
We may avoid putting prefix `left_` and `right_` for now as these keypoints are not mapped to human_data.

Reply:
Ok sure, we can discuss about the GTA convention in the future